css/layout/sidebar.css

```diff
@@ -24,6 +24,11 @@
 	padding: 1rem 0.2rem;
 	overflow-y: auto;
 }
+.sidebar__links {
+	text-decoration: none;
+	color: inherit;
+	padding: 1rem 8rem;
+}
```

Comment:
if it's a tag then you can use inherit and decoration property in reset css

Reply:
Done